openhands-cli/README.md

```diff
@@ -0,0 +1,45 @@
+# OpenHands CLI
+
+A lightweight CLI/TUI to interact with the OpenHands agent (powered by agent-sdk). Build and run locally or as a single executable.
+
+## Quickstart
+
+- Prerequisites: Python 3.12+, curl
+- Install uv (package manager):
+  ```bash
+  curl -LsSf https://astral.sh/uv/install.sh | sh
+  # Restart your shell so "uv" is on PATH, or follow the installer hint
+  ```
+
+### Run the CLI locally
+```bash
+# Install dependencies (incl. dev tools)
+make install-dev
+
+# Optional: install pre-commit hooks
+make install-pre-commit-hooks
+
+# Start the CLI
+make run
+# or
+uv run openhands-cli
+```
+
+Tip: Set your model key (one of) so the agent can talk to an LLM:
+```bash
+export OPENAI_API_KEY=...
+# or
+export LITELLM_API_KEY=...
```

Comment:
```suggestion
```

Reply:
Maybe just a file view to the commit before this merge? Maybe the tag of latest release